welcome/index.adoc

```diff
@@ -300,7 +300,7 @@ Use the Cluster Version Operator (CVO) to upgrade your {product-title} cluster.
 - **xref:../otel/otel-installing.adoc#install-otel[Red Hat build of OpenTelemetry]**: Instrument, generate, collect, and export telemetry traces, metrics, and logs to analyze and understand your software's performance and behavior. Use open source backends like Tempo or Prometheus, or use commercial offerings. Learn a single set of APIs and conventions, and own the data that you generate.
 
 // CHECK IN CLOSER TO THE GA DATE BECAUSE FEATURE WORK IS NOT MERGED; ESTIMATED MERGE BY 19 FEB. ADD xref:../network_observability/metrics-alerts-dashboards.adoc#network-observability-viewing-dashboards_metrics-dashboards-alerts to [view dashbaords text.
-- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can view dashboards and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
+- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.html[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.
```

Comment:
- **xref:../network_observability/network-observability-overview.adoc#network-observability-overview[Network Observability]**: Observe network traffic for {product-title} clusters by using eBPF technology to create and enrich network flows. You can xref:../network_observability/metrics-alerts-dashboards.adoc#<anchor_id>_<context>
[view dashboards, customize alerts], and xref:../network_observability/observing-network-traffic.adoc#network-observability-trafficflow_nw-observe-network-traffic[analyze network flow] information for further insight and troubleshooting.